packages/components/package.json

```diff
@@ -21,11 +21,12 @@
         "@pinecone-database/pinecone": "^0.0.12",
         "axios": "^0.27.2",
         "chromadb": "^1.3.1",
+        "cohere-ai": "^6.2.0",
         "d3-dsv": "2",
         "dotenv": "^16.0.0",
         "express": "^4.17.3",
         "form-data": "^4.0.0",
-        "langchain": "^0.0.53",
+        "langchain": "^0.0.56",
```

Comment:
Thanks @mileszim for the PR!

Is `0.0.56` needed for cohere-ai? If not, can you downgrade to `^0.0.53` until we do a full regression test on `0.0.56`?

Reply:
Checked it works with ^0.0.53, temporarily downgraded